plugins/Diagnostics/lang/en.json

```diff
@@ -1,6 +1,7 @@
 {
     "Diagnostics": {
         "ConfigFileTitle": "Config file",
+        "MysqlTemporaryTablesWarning": "MySQL Permission CREATE TEMPORARY TABLES is required for Matomo to work properly.",
```

Comment:
could we maybe link to [this FAQ](https://matomo.org/faq/how-to-install/faq_23484/)?
refs  #15237

Reply:
done